docs/benchmarkdotnet.md

```diff
@@ -367,7 +367,7 @@ It's possible to benchmark a private build of .NET Runtime. You just need to pas
 So if you made a change in CLR and want to measure the difference, you can run the benchmarks with:
 
 ```cmd
-dotnet run -c Release -f net472 -- --clrVersion $theVersion
+dotnet run -c Release -f net7.0 -- --clrVersion $theVersion
```

Comment:
this particular example refers to running a local build of .NET Framework so we need to use the old moniker

```suggestion
dotnet run -c Release -f net48 -- --clrVersion $theVersion
```